department-of-veterans-affairs/caseflow
# Restoring changes reverted due to Hearing Prep Prod Issue

Resolves #8990 , #9137 

### Description
Changes made in PR #9137 had to be reverted due to a Hearing Prep load issue. This PR includes all changes reverted and the fix for Hearing Prep.

### Acceptance Criteria
- [ ] Refactored tests pass

### Testing Plan
1. Go to Hearing Prep
2. Ensure the page loads
